### ISSUE TITLE
JAWS screen reader Virtual PC cursor does not allow selection

### ISSUE THREAD

**Issue by devinbinnie** [electron/electron#20384]
### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 2.x, 4.x, 5.x
* **Operating System:**
  * Windows 10 1903
* **Last Known Working Electron version:**
  * None that I know of

### Expected Behavior
When using JAWS' Virtual PC cursor to navigate through a document, pressing SHIFT+Down while a block is highlighted should select the highlighted text such that the text can be copied with Ctrl+C.

### Actual Behavior
The text is not selected

### To Reproduce
1) Open the Electron API demos app
2) Click on Getting Started
3) Open JAWS, and enable Virtual PC cursor
4) Once enabled, use it to navigate to a text block in the RHS (for example, `BrowserWindow`)
5) Once highlighted on a text block, press SHIFT+Down
6) The text should be selected.

### Additional Information
This is doable in Chrome on Windows 10, but does not work in the Electron versions mentioned above.
Bug filed with JAWS: https://github.com/FreedomScientific/VFO-standards-support/issues/321

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.

**Comment by devinbinnie** [comment]
@electron-triage Looks like it's still reproducing on Electron 6. Same steps as above.
This was tested using Mattermost v4.4, available here: https://github.com/mattermost/desktop/releases
